internal/internal_workflow_testsuite.go

```diff
@@ -1044,6 +1044,11 @@ func (env *testWorkflowEnvironmentImpl) RequestCancelWorkflow(domainName, workfl
 	return nil
 }
 
+func (env *testWorkflowEnvironmentImpl) SignalExternalWorkflow(domainName, workflowID, runID, signalName string, input []byte, callback resultHandler) {
+	env.signalHandler(signalName, input)
```

Comment:
this is not correct. env.signalHandler only send signal to the current workflow, not to external workflow.

Reply:
I need some advise. 
Currently there is no easy way to unit test interaction between 2 workflow. So I mimic the signal process to external workflow and set future. 
Can I add a cadence sample and do the integration test that way?